adlfs/spec.py

```diff
@@ -1278,6 +1278,31 @@ async def _rm_files(
 
     sync_wrapper(_rm_files)
 
+    async def _rm_file(
+        self, path: typing.Union[str, typing.List[str]], delimiter: str = "/", **kwargs
```

Comment:
And on a similar note, I'm not sure if we should be exposing a `delimiter` for this method since there is not really any recursive nature to this public contract and instead always just use `/` if we need any splitting logic. Also not supporting a `delimiter` seems consistent with the other implementations I linked.